config/migrate/migrate.go

```diff
@@ -3,18 +3,33 @@ package migrate
 
 import (
 	"fmt"
+	"io"
 	"os"
 	"path/filepath"
+	"strings"
 
+	"github.com/mitchellh/go-homedir"
 	"github.com/qri-io/ioes"
+	"github.com/qri-io/qfs/qipfs"
 	"github.com/qri-io/qri/config"
 	qerr "github.com/qri-io/qri/errors"
 )
 
+// ErrNeedMigration indicates a migration is required
+var ErrNeedMigration = fmt.Errorf("migration required")
+
 // RunMigrations checks to see if any migrations runs them
-func RunMigrations(streams ioes.IOStreams, cfg *config.Config) (err error) {
+func RunMigrations(streams ioes.IOStreams, cfg *config.Config, interactive bool) (err error) {
 	if cfg.Revision != config.CurrentConfigRevision {
-		streams.PrintErr("migrating configuration...")
+		if interactive {
+			msg := `Your repo needs updating before qri can start. 
+Run migration now?`
+			if !confirm(streams.Out, streams.In, msg) {
+				return qerr.New(ErrNeedMigration, `your repo requires migration before it can run`)
```

Comment:
Something clearer.

'Your repo needs to be migrated before any qri commands can be run.'

Or, from line 25:

'Your repo needs updating before qri can start.'

They should probably say the same thing.